examples/offline_inference_tpu.py

```diff
@@ -19,10 +19,12 @@
 
 # Set `enforce_eager=True` to avoid ahead-of-time compilation.
 # In real workloads, `enforace_eager` should be `False`.
-llm = LLM(model="google/gemma-2b", enforce_eager=True)
+llm = LLM(model="neuralmagic/Meta-Llama-3.1-8B-Instruct-quantized.w8a8",
+          enforce_eager=True,
+          max_model_len=1024)
 outputs = llm.generate(prompts, sampling_params)
 for output, answer in zip(outputs, answers):
     prompt = output.prompt
     generated_text = output.outputs[0].text
     print(f"Prompt: {prompt!r}, Generated text: {generated_text!r}")
-    assert generated_text.startswith(answer)
```

Comment:
does this no longer work?

Reply:
The text was hardcoded from a previous model

Reply:
should we update the `answers` list in this file then? or remove it?

Reply:
To be removed before merge, just using this file for sanity check in dev